docs/commands/bundle.md

```diff
@@ -1,117 +1,91 @@
 
-```mermaid
-graph LR
-    preprocessors[Preprocessors]
-    rules[Rules]
-    decorators[Decorators]
-    preprocessors --> rules --> decorators
-```
-
-### `bundle` usage
-
-```
-Positionals:
-  entrypoints                                              [array] [default: []]
-Options:
-  --version            Show version number.                            [boolean]
-  --help               Show help.                                      [boolean]
-  --lint               Lint definitions               [boolean] [default: false]
-  --output, -o                                                          [string]
-  --format             Use a specific output format.
-                        [choices: "stylish", "codeframe"] [default: "codeframe"]
-  --max-problems       Reduce output to max N problems.  [number] [default: 100]
-  --ext                Bundle file extension.   [choices: "json", "yaml", "yml"]
-  --skip-rule          Ignore certain rules.                             [array]
-  --skip-preprocessor  Ignore certain preprocessors.                     [array]
-  --skip-decorator     Ignore certain decorators.                        [array]
-  --dereferenced, -d   Produce fully dereferenced bundle.              [boolean]
-  --force, -f          Produce bundle output even when errors occur.   [boolean]
-  --config             Specify path to the config file.                 [string]
-```
+:::success Tip
+To learn more about preprocessors, rules, and decorators, please refer to [this](../custom-rules.md) document.
+:::
 
-The command:
+## Usage
 
 ```bash
-openapi bundle --output <outputName> --ext <ext> [entrypoints...]
-```
-
-- `[entrypoints...]` corresponds to the name(s) of your root document(s).
-- Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
-- The `<outputName>` is your desired output filename or folder. If the folder doesn't exist, it's automatically created.
-- Use `--ext <ext>` to specify the type and extension of the output file: `.json`, `.yml` or `.yaml`.
+openapi bundle <entrypoints>...
+openapi bundle <entrypoints> [--max-problems=<n>]
+openapi bundle <entrypoints> [--lint] [--config=<path>]
+openapi bundle <entrypoints>... -o <outputName> --ext <ext>
+openapi bundle --version
+```
+
+## Options
+
+option                 | type      | required?    | default     | description
+-----------------------|:---------:|:------------:|:-----------:|------------
+`entrypoints`          | `array`   | yes          | `[]`        | Array of API root definition filenames that need to be bundled. Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
+`--config`             | `string`  | no           | -           | Specify path to the config file
+`--dereferenced`, `-d` | `boolean` | no           | -           | Generate fully dereferenced bundle
```

Comment:
suggestion: You asked a really good question in Slack about what "fully dereferenced" means. I think it would be useful to explain that here because new users who are not familiar with this terminology will likely be confused about it, too. So it would help to have it explained; a sentence or two should be just enough.